From 1657a661d4532b9c02a1a81d25314140ebbdb311 Mon Sep 17 00:00:00 2001
From: Zhantao Tang <zhantao.tang@windriver.com>
Date: Wed, 16 Jun 2021 17:18:12 +0800
Subject: [PATCH] dts: s32g274a-rdb2: disable ARQ107 phy node explicitly to fix
 error message when resume

On RDB2, there is an phy node ARQ107 connected with PFE EMAC1,
and PFE EMAC1 is disabled by default, so the ARQ107 phy node
should be disabled explicitly too, or else there will be error
message like following when resume from STR:

	PM: dpm_run_callback(): mdio_bus_phy_resume+0x0/0xa4 returns -19
	PM: Device stmmac-0:03 failed to resume: error -19

Upstream-Status: Pending

Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
Signed-off-by: Bruce Ashfield <bruce.ashfield@gmail.com>
---
 arch/arm64/boot/dts/freescale/fsl-s32g274a-rdb2.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-s32g274a-rdb2.dts b/arch/arm64/boot/dts/freescale/fsl-s32g274a-rdb2.dts
index 60cac789b7be..91c82cd28783 100644
--- a/arch/arm64/boot/dts/freescale/fsl-s32g274a-rdb2.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-s32g274a-rdb2.dts
@@ -58,6 +58,7 @@ mdio_a_phy3: ethernet-phy@3 {
 		#address-cells = <1>;
 		#size-cells = <0>;
 		reg = <3>;
+		status = "disabled";
 	};
 };
 
-- 
2.25.1

